sdk/src/config/DEVNET.ts

```diff
@@ -0,0 +1,95 @@
+import { CONTRACTS, Network as Environment } from '@certusone/wormhole-sdk';
+import { ChainConfig, Context, Contracts, WormholeConfig } from '../types';
+
+/**
+ * devnet chain name to chain id mapping
+ */
+export const DEVNET_CHAINS = {
+  ethereum: 2,
+  terra2: 18,
+  osmosis: 20,
+  wormchain: 3104,
+} as const;
+
+/**
+ * devnet chain name type
+ */
+export type DevnetChainName = keyof typeof DEVNET_CHAINS;
+/**
+ * devnet chain id type
+ */
+export type DevnetChainId = (typeof DEVNET_CHAINS)[DevnetChainName];
+/**
+ * chain name to contracts mapping
+ */
+export type ChainContracts = {
+  [chain in DevnetChainName]: Contracts;
+};
+
+const DEVNET: { [chain in DevnetChainName]: ChainConfig } = {
+  ethereum: {
```

Comment:
Why is Ethereum in Devnet?  Also, Solana is a devnet chain, does it belong in here?  What's the purpose of creating a devnet file instead of using testnet?

Reply:
I think this is for the Tilt environment from the wormhole monorepo

Reply:
As @kev1n-peters mentioned. There was a point I needed to set up the tilt environment locally, which required defining new chains and RPCs. I can remove it if you think it muddles up the plugin.

Re: Solana, their testnet is called "devnet"